jwst/tweakreg/tweakreg_step.py

```diff
@@ -170,17 +170,35 @@ def process(self, input):
             yoffset=self.yoffset
         )
 
-        align_wcs(
-            imcats,
-            refcat=None,
-            enforce_user_order=self.enforce_user_order,
-            expand_refcat=self.expand_refcat,
-            minobj=self.minobj,
-            match=tpmatch,
-            fitgeom=self.fitgeometry,
-            nclip=self.nclip,
-            sigma=(self.sigma, 'rmse')
-        )
+        try:
+            align_wcs(
+                imcats,
+                refcat=None,
+                enforce_user_order=self.enforce_user_order,
+                expand_refcat=self.expand_refcat,
+                minobj=self.minobj,
+                match=tpmatch,
+                fitgeom=self.fitgeometry,
+                nclip=self.nclip,
+                sigma=(self.sigma, 'rmse')
+            )
+
+        except ValueError as e:
+            msg = e.args[0]
+            if (msg == "Too few input images (or groups of images) with "
+                "non-empty catalogs."):
+                # we need at least two exposures to perform image alignment
+                self.log.warning(msg)
+                self.log.warning("At least two exposures are required for "
+                                 "image alignment.")
+                self.log.warning("Nothing to do. Skipping 'TweakRegStep'...")
+                self.skip = True
+                for model in images:
+                    model.meta.cal_step.tweakreg = "SKIPPED"
+                return input
```

Comment:
This should probably return `images`, right?

Reply:
No, `input` is perfect.

Reply:
Yes, you are right.